config/api.yml

```diff
@@ -1792,10 +1792,14 @@
     :collection_actions:
       :get:
       - :name: read
-        :identifier: ems_infra_show_list
+        :identifier:
+        - ems_infra_show_list
+        - ems_block_storage_show_list
```

Comment:
Are Block Storage Managers providers ?  i.e. subclass of ExtManagementSystem or Provider ? If so then this change is fine, i.e.  User is authorized to see both variants as per the miq product feature selections and should be here since they are both accessible via /api/providers.

Reply:
yes, exactly, it is:
```
irb(main):013:0> ExtManagementSystem > ManageIQ::Providers::StorageManager
true
```